esp/src/eclwatch/GridDetailsWidget.js

```diff
@@ -114,9 +114,9 @@ define([
             var MyMemory = declare("MyMemory", [Memory], {
                 idProperty: this.idProperty,
                 data: [],
-                setData: function (data) {
+                setData: function (data, noDataMessage) {
                     var retVal = this.inherited(arguments);
-                    context.setGridNoDataMessage(context.i18n.noDataMessage);
+                    context.setGridNoDataMessage(noDataMessage);
```

Comment:
This should be:  `context.setGridNoDataMessage(noDataMessage || context.i18n.noDataMessage);`

Reply:
Changes applied and tested.